### ISSUE TITLE
Missing descriptive alt text, needed for accessibility

### ISSUE THREAD

**Issue by discospiff** [JacobJospehj2/IS7024-001#20]
https://github.com/JacobJospehj2/IS7024-001/blob/01bd0259154ccb98eb9e8f6b20fc1804ffa2528a/IS7024FinalProjectBreweryArc/BreweryArc/Pages/BrewerySingle.cshtml#L140
